src/key/user.js

```diff
@@ -157,11 +157,12 @@ class User {
 
   /**
    * Verify User. Checks for existence of self signatures, revocation signatures
-   * and validity of self signature. Throws when there are no valid self signatures.
+   * and validity of self signature.
    * @param  {SecretKeyPacket|
    *          PublicKeyPacket} primaryKey The primary key packet
    * @param  {Date}            date       Use the given date instead of the current time
    * @returns {Promise<true>}             Status of user
```

Comment:
It's not super important, but maybe you want to change this one as well?

Reply:
I wouldn't change this for now, because there is `verifyCertificate` that it is somewhat related (it's used in a similar way in `verifyAllCertifications` or `verifyAllUsers`), and that one needs to return `null | true`, hence I think it's best to uniform the two together eventually.